pages/en/lb2/Model-definition-JSON-file.md

```diff
@@ -101,6 +101,13 @@ Properties are required unless otherwise designated.
       </td>
     </tr>
     <tr>
+      <td>forceId</td>
+      <td>Boolean</td>
+      <td>
+        The default value is set to true whenever a model uses an auto-generated ID. If it is set to true it prevents users from setting the auto-id value manually.
```

Comment:
IIRC, this applies to LoopBack 3.x applications only, doesn't it? I think LoopBack 2.x always defaults `forceId` to `false`.

My point is that this change should be made in `pages/en/lb3/Model-definition-JSON-file.md`, not `pages/en/lb2/Model-definition-JSON-file.md`.